application/src/main/java/org/thingsboard/server/actors/ruleChain/DefaultTbContext.java

```diff
@@ -533,6 +534,11 @@ public RuleEngineDeviceProfileCache getDeviceProfileCache() {
         return mainCtx.getDeviceProfileCache();
     }
 
+    @Override
+    public RuleEngineAssetProfileCache getAssetProfileCache() {
```

Comment:
Please take a look at methods: 'assetCreatedMsg' and 'alarmActionMsg'. They need to be modified and use the correct queueName and ruleChainId (similar to implementation that is related to 'device' type).